fuse-springboot/fuse-springboot-bom/pom.xml

```diff
@@ -32,6 +32,15 @@
 
     <dependencyManagement>
         <dependencies>
+            <!-- Bump jackson to versions newer than Spring Boot curated ones in order to prevent some CVEs -->            
+            <dependency>
+                <groupId>com.fasterxml.jackson</groupId>
+                <artifactId>jackson-bom</artifactId>
+                <version>2.9.10.20191020</version>
```

Comment:
Is there a specific reason to use this version?

Reply:
This version of the jackson-bom is including a curated set of jackson versions including jackson-databind 2.9.10.1.
jackson-databind 2.9.10.1 is fixing those CVEs, and also this is the databind version on which most project are aligned in 7.6.